demos/http/http_demo_mutual_tls/demo_config.h

```diff
@@ -50,24 +50,41 @@
 /************ End of logging configuration ****************/
 
 /**
- * @brief HTTP server endpoint for your AWS IoT Core.
+ * @brief Your AWS IoT Core endpoint.
  *
- * @note This can be found in the "Settings" section of AWS IoT Core
+ * @note Your AWS IoT Core endpoint can be found in the AWS IoT console under
+ * Settings/Custom Endpoint, or using the describe-endpoint API.
+ * @ref https://docs.aws.amazon.com/iot/latest/apireference/API_DescribeEndpoint.html
  */
-#define IOT_CORE_ENDPOINT                 "placeholder.iot.us-west-2.amazonaws.com"
+#define IOT_CORE_ENDPOINT                 "your-aws-iot-core-endpoint"
+
+/**
+ * @brief ALPN protocol name to be sent as part of the ClientHello message.
+ *
+ * @note When using ALPN, port 443 must be used to connect to AWS IoT Core.
+ */
+#define IOT_CORE_ALPN_PROTOCOL_NAME       "\x0ex-amzn-http-ca"
 
 /**
  * @brief AWS IoT Core server port number for HTTPS connections.
  *
- * @note Port 443 is normally used for HTTPS connections. However, because 443
- * is reserved for MQTT in AWS IoT Core, port 8443 is used instead.
+ * For this demo, an X.509 certificate is used to verify the client.
+ * However, it is also possible to use Signature Version 4 as detailed here:
```

Comment:
I wouldn't include lines 72&73 yet since (AFAIK) this library does not support SigV4 authentication.

Reply:
We will never add Sigv4 comments into this demo. Sigv4 is needed for AWS IoT only when doing a request to the non-account-specific AWS IoT endpoint for an AWS API command like: `GET /endpoint?endpointType=endpointType HTTP/1.1`. When making requests to general AWS service endpoints we would be initially connecting over TLS with just server authentication.